packages/jaeger-ui/src/utils/prefix-url.test.js

```diff
@@ -37,3 +38,19 @@ describe('prefixUrl()', () => {
     });
   });
 });
+
+describe('regExEscape()', () => {
+  const tests = [
+    { source: undefined, target: '' },
+    { source: null, target: '' },
+    { source: `${IPV6_HOST_ADDRESS}`, target: `${IPV6_HOST_ADDRESS}` },
+    { source: `[${IPV6_HOST_ADDRESS}]`, target: `\\[${IPV6_HOST_ADDRESS}\\]` },
+    { source: `[${IPV6_HOST_ADDRESS}]:8080`, target: `\\[${IPV6_HOST_ADDRESS}\\]:8080` },
+  ];
+
+  tests.forEach(({ source, target }) => {
+    it(`regexEscape "${source}" correctly`, () => {
+      expect(regExEscape(source)).toBe(target);
```

Comment:
I think a better test would be to move the calculation of pathPrefix into a function and test that function with the permutation of addresses. Testing regExEscape is too low level and unnecessarily exposes implementation detail from the util file.

Reply:
I agree. Have modified the tests.